sound/soc/sof/pm.c

```diff
@@ -324,6 +324,9 @@ static int sof_suspend(struct device *dev, int runtime_suspend)
 	if (!sdev->ops->suspend)
 		return 0;
 
+	/* release trace */
+	snd_sof_release_trace(sdev);
+
```

Comment:
is this sequence change really important? 
I think sof_suspend_streams is only for those topology defined pipelines, so make sure snd_sof_release_trace() is called before dsp_suspend() should be enough.

can you try keep the place where it is but only change from snd_sof_dma_trace_release() to  snd_sof_release_trace()?

Reply:
sorry, better to make it before SOF_IPC_PM_CTX_SAVE ipc sending.